src/libecalc/presentation/yaml/yaml_types/yaml_stream_conditions.py

```diff
@@ -25,16 +25,36 @@ class YamlRate(YamlTimeSeries):
 class YamlEmissionRate(YamlTimeSeries):
     model_config = ConfigDict(title="Rate")
 
-    unit: Literal[Unit.KILO_PER_DAY, Unit.TONS_PER_DAY] = Unit.KILO_PER_DAY
+    unit: str = Unit.KILO_PER_DAY.name
```

Comment:
```suggestion
    unit: Literal[Unit.KILO_PER_DAY.name, Unit.TONS_PER_DAY.name] = Unit.KILO_PER_DAY.name
```

This will keep the validation in the json-schema I think. We can then remove `validate_unit`. 

If the custom error message is needed we should check if pydantic supports a custom error message directly, or we can introduce a 'before' validator. The last option will duplicate the validation logic, so it shouldn't be the first choice.

Reply:
Have tried, did not work. As discussed: adding separate unit enum class, removing vaidate_unit